Documentation/doc/scripts/testsuite.py

```diff
@@ -74,7 +74,7 @@ def write_report(args):
 </head><body>
 <h1 id="maintitle">Doxygen Manual Results</h1>'''
     page_footer='''<table border="1" cellspacing="2" cellpadding="5" class="test-results">
-    <tr><td/><th colspan="3">Doxygen 1.8.4</th><th colspan="3">Doxygen 1.8.13(official)</th><th colspan="3">Doxygen master</th></tr>
+    <tr><td/><th colspan="3">Doxygen 1.8.13(official)</th><th colspan="3">Doxygen 1.8.18</th><th colspan="3">Doxygen master</th></tr>
```

Comment:
official is the second one then, right?

Reply:
I am surprised: this PR is about the transition from Python2 to Python3, but it seems that, at the same same we have something about the transition from Doxygen 1.8.13 to Doxygen 1.8.18... (and why version 1.8.18 by the way?!).